plugins/Annotations/Controller.php

```diff
@@ -114,6 +119,8 @@ public function saveAnnotation()
             // save the annotation
             $view->annotation = Request::processRequest("Annotations.save");
 
+            $view->users = $this->getAnnotationUsers();
```

Comment:
Any reason for calling this without a parameter? If you want to reset `$view->users`, simply set it to an empty array.